plugins/orchestrator/src/dataFormatters/DataFormatterInterface.ts

```diff
@@ -0,0 +1,5 @@
+interface DataFormatterInterface<Data, FormattedData> {
```

Comment:
```suggestion
interface DataFormatter<Data, FormattedData> {
```
I think the `Interface` suffix is redundant here. Also applies to the filename.